src/plugins/cordova-plugin-inappbrowser/app-host-handlers.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+module.exports = function () {
+    var Browser = require('./inapp-browser'),
+        instance;
+
+    function execute(fnName) {
+        if (!instance) {
+            return;
+        }
+
+        var fn = instance[fnName],
+            args;
+
+        if (Object.keys(arguments).length > 1) {
+            // remove the first element since it is the "fnName" parameter
+            Array.prototype.shift.apply(arguments);
+            args = arguments;
+        }
+
+        return fn.apply(instance, args);
+    }
+
+    return {
+        'InAppBrowser': {
```

Comment:
Ignore my previous comment, I was thinking about the JS interface, while this is the exec bridge interface. 👍